common/openapi.yml

```diff
@@ -653,6 +660,8 @@ components:
           format: date-time
         last_run:
           $ref: '#/components/schemas/MethodLastRunDetails'
+        source_details:
+          $ref: '#/components/schemas/GithubMethodSourceDetailsResponse'
```

Comment:
Why `source_details` and not `method_details`?

Reply:
What if the method is not a github method?

Reply:
I named this is as `source_details` keeping in mind that we may have other sources. However, what is returned here will vary, so we'll want to keep that in mind when we support other sources.